hack/compare-k8s-admission-plugins.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
Adapt https://github.com/gardener/gardener/blob/master/docs/development/new-kubernetes-version.md#adapting-gardener with information about this script.